src/main/java/org/elasticsearch/index/query/FuzzyQueryBuilder.java

```diff
@@ -62,6 +62,14 @@ public FuzzyQueryBuilder(String name, Object value) {
     }
 
     /**
+     * package private constructor for serialization only
+     */
+    FuzzyQueryBuilder() {
+        this.name = null;
+        this.value = null;
```

Comment:
sounds good, we were even thinking about merging those, we will not move them to separate packages for sure.